### PR TITLE
remember_schema_config

### PR DESCRIPTION
**Pull request by jcieslar**

### CODE REVIEW

File: app/helpers/dataclips/application_helper.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def find_and_display_insight(clip_id, params = {}, options = {}, &block)
 
     # insight options: time_zone, per_page, connection, schema
     insight = Dataclips::Insight.get!(clip_id, params, options.slice('time_zone', 'per_page', 'connection', 'schema'))
-    display_insight(insight, {limit: options['limit']}, &block)
+    display_insight(insight, {limit: options['limit'], save_schema_config: options['save_schema_config']}, &block)
```
Comment (defkode) [inline app/helpers/dataclips/application_helper.rb]: save_schema_config or remember_schema_config ?

File: app/helpers/dataclips/application_helper.rb [hunk 2]
```diff
@@ -28,6 +28,7 @@ def dataclips_insight_config(insight, options = {})
 
     schema = load_dataclip_insight_schema(insight)
     schema_md5 = Digest::MD5.hexdigest(Marshal.dump(schema.to_json))
+    schema_md5 = nil if options['remember_schema_config'] == false
```
Comment (defkode) [inline app/helpers/dataclips/application_helper.rb]:
```suggestion
    if options['remember_schema_config'] 
       schema_md5 = Digest::MD5.hexdigest(Marshal.dump(schema.to_json))
    end
```

Comment (jcieslar) [reply app/helpers/dataclips/application_helper.rb]: if we do not have `remember_schema_config ` then we will not remember schema in your example. I would like to have  `remember_schema_config` default as true

Comment (defkode) [reply app/helpers/dataclips/application_helper.rb]: then you should write in the beginning of method:

```rb
remember_schema_config = options.fetch('remember_schema_config', true)

**Pull request merged**